R/extend_scenario_trajectory.R

```diff
@@ -109,3 +110,154 @@ extend_scenario_trajectory <- function(data,
       .data$scenario_geography, .data$ald_sector, .data$technology, .data$year
     )
 }
+
+
+#' Extend the scenario pathways based on the fair share approach (now known as
+#' market share approach).  We first ensure that all scenarios are extended to
+#' the same point in time (start of the analysis plus forecast time frame.
+#' Then we take the first value of the production on company level as the start
+#' value and from the second year onward, we calculate the level of production
+#' for a company under the scenarios at hand by applying the relative changes
+#' for the respective years to the absolute values of the previous year.
+#'
```

Comment:
please make more specifc here how it extends extend_scenario_trajectory

Reply:
overhauled